README.md

```diff
@@ -58,46 +58,27 @@ Extend @ni/eslint-config/typescript in the ESLint configuration. Configure the @
 ### Angular
 
 ESLint support for Angular is provided by [@angular-eslint](https://github.com/angular-eslint/angular-eslint#readme). It's recommended to use @angular-eslint/schematics to
-configure ESLint for Angular projects especially when migrating from TSLint.
-
-[Use version 1.x.x](https://github.com/angular-eslint/angular-eslint#supported-angular-cli-versions) for Angular versions less than 11.1.0.
-
-#### Generating a Workspace
-
-Workspaces [generated with @angular-eslint](https://github.com/angular-eslint/angular-eslint#quick-start-with-angular-and-eslint) will generate projects with @angular-eslint.
-
-```bash
-> npm i -g @angular/cli @angular-devkit/core @angular-devkit/schematics @angular-eslint/schematics
-> ng new --collection=@angular-eslint/schematics
-```
-
-Extend @ni/eslint-config/typescript in .eslintrc.json for TypeScript and templates.
-```json
-"overrides": [{
-    "files": ["*.ts"],
-    ...
-    "extends": ["@ni/eslint-config/typescript"]
-},
-    ...
-]
-```
-
-#### Migrating a Workspace
-
-1. Install @angular-eslint/schematics and @ni/eslint-config.
+configure ESLint for Angular projects especially when migrating from TSLint. [Use version 1.x.x](https://github.com/angular-eslint/angular-eslint#supported-angular-cli-versions) for Angular versions less than 11.2.0.
 
+1. [Use the schematic](https://github.com/angular-eslint/angular-eslint#quick-start-with-angular-and-eslint) to add ESLint to new workspaces, and new applications and libraries will be generated with ESLint as well.
     ```bash
-    npm install --save-dev @angular-eslint/schematics @ni/eslint-config.
+    > ng add @angular-eslint/schematics
+    > ng g @angular-eslint/schematics:convert-tslint-to-eslint --remove-tslint-if-no-more-tslint-targets --ignore-existing-tslint-config
     ```
-
-2. Follow the @angular-eslint [instructions](https://github.com/angular-eslint/angular-eslint#migrating-an-angular-cli-project-from-codelyzer-and-tslint) to run the schematic, listed here for convenience.
-
+2. Extend @ni/eslint-config/typescript in .eslintrc.json for TypeScript and templates.
```

Comment:
Can we consistently use backtick escapes for npm package names and filenames? If you don't mind a quick pass through the README since we are not consistent right now.

Reply:
Sure, done.

Reply:
Comment in the meeting to replace three ticks with one tick for inline uses

Reply:
Did a quick pass to clean it up. Not resetting reviewers since it is a minor change but if there are concerns let me know and can do a follow-up.